### PR TITLE
Configure Git Workflows #4

### PR DESCRIPTION
**Pull request by Uticodes**

### CODE REVIEW

File: .github/workflows/pr-validation.yml [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+        run: |
+          mkdir -p junit
+          find . -type f -regex ".*/build/test-results/.*xml" -exec cp {} junit/ \;
+      - name: Upload test results
+        if: always()
+        uses: actions/upload-artifact@v1
+        with:
+          name: junit-results
+          path: junit
+  instrumented-tests:
```
Comment (princessdharmy) [inline .github/workflows/pr-validation.yml:89]: Kindly take this out as we may not be doing any UI tests for now. The process for this takes too long to complete.

Comment (Uticodes) [reply .github/workflows/pr-validation.yml:89]: okay

File: .github/workflows/pr-validation.yml
Comment (princessdharmy) [inline .github/workflows/pr-validation.yml:1]: - I don't know why you added PR validation. Could you shed more light on that?

- The `Enforce Label` workflow should be in a separate file

Comment (Uticodes) [reply .github/workflows/pr-validation.yml:1]: Why I added Pr validation was to validate the branch name and Title.
I saw it as one of the recommended workflows.

Comment (princessdharmy) [reply .github/workflows/pr-validation.yml:1]: Thanks for the feedback. If I may ask further, what is the essence of validating a branch name and title?
Also, I think the name of the workflow should not be `PR Validation`. 

Suggestion:
- If you still want to include PR Validation, then, put it in a separate file.
- For others, I would suggest `Build and Test`. Then, the file name could be `mainBuild.yml`

Comment (Uticodes) [reply .github/workflows/pr-validation.yml:1]: 1. Okay, for instance, in some projects or organizations, they validate PRs using the ticket title and number. maybe Jira ticket or GitHub issues, so they might want to be sure that the PR carries the ticket or issue number on it
But I don't think we need it here,, so I'll remove it.
2.  I'll rename the name name and the file name.

**Pull request merged**